frontend/src/metabase/actions/containers/ActionCreator/ActionCreatorView.tsx

```diff
@@ -125,6 +125,7 @@ export default function ActionCreatorView({
               parameters={action.parameters ?? []}
               formSettings={formSettings}
               isEditable={isEditable && canChangeFieldSettings}
+              isPublic={!!action.public_uuid}
```

Comment:
nit: `action.public_uuid != null`

Reply:
<img width="513" alt="image" src="https://github.com/metabase/metabase/assets/125459446/711bee42-99a1-4f3f-9e53-dfa79ce4d5ae">

Reply:
ah, I see, you used `!=`, I used to have explicit checks on null and undefined